application/django/README.md

```diff
@@ -1,24 +1,38 @@
+# Django application in Docker
+This example checks if the environment variables are set with SecretHub run within the Django application. If this is successful, you will receive a code 200 and a Welcome <username> on localhost:8080, if not a code 500 is shown. 
+
+## Prerequisites
+1. Docker installed and running
+2. SecretHub installed
```

Comment:
We could be the user of service by linking `SecretHub` and `Docker` to their respective install pages.